micro_sam/util.py

```diff
@@ -109,35 +113,9 @@ def models():
             "vit_b_em": "https://zenodo.org/record/8250260/files/vit_b_em.pth?download=1",
         },
     )
-    # This extra dictionary is needed for _get_checkpoint to download weights from a specific checkpoint
```

Comment:
We don't need this anymore. We just use the keys in the registry as filenames in the cache.